addons/binding/org.openhab.binding.amazondashbutton/README.md

```diff
@@ -99,7 +99,9 @@ There is one supported Thing, the "Amazon Dash Button".
 
 ## Discovery
 
-Background discovery is not supported as it is not possible to distinguish between a Dash Button and other Amazon devices like the Kindle, a Fire TV or an Echo speaker.
+Background discovery is not supported as it is not possible to distinguish
+between a Dash Button and other Amazon devices like the Kindle,
+a Fire TV or an Echo speaker.
```

Comment:
Consider changing to "Background discovery is not supported, as it is not possible to distinguish between a Dash Button and some other type of Amazon devices such as the Kindle reader, a Fire TV, or an Echo speaker."